app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainActivityPanel.java

```diff
@@ -101,7 +112,16 @@ public boolean isSlidingPanelOpen() {
         return PanelViewModel.isOpenState(mSlidingLayout.getPanelState());
     }
 
+    private KeyboardVisibilityEventListener createKeyboardVisibilityEventListener() {
+        return isOpen -> adjustPanelDimensions();
+    }
+
+    private void adjustPanelDimensions() {
+        mPanelListener.onPanelSlide(null, mSlidingLayout.getCurrentParallaxOffset());
```

Comment:
You shouldn't self invoke an event, maybe extract to another method instead.
Why do you use parallaxOffset? It meant to be used for something else in the panel.